src/asim/configs/crossborder/settings.yaml

```diff
@@ -4,7 +4,7 @@ inherit_settings: False
 
 # number of households to simulate
 
-households_sample_size: 0
+households_sample_size: 101343
```

Comment:
Reset households_sample_size

Reply:
I think that may be set to the correct value before the final iteration, but it's good to set it to zero just in case (I just committed that)